account/account.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package account
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/network"
+
```

Comment:
nit: remove space. We should the `gci` formatter, perhaps with the same settings as avalanchego.

Reply:
ran `gofumpt` and no changes were made by `gofumpt`